aspnet/dnx/projects.rst

```diff
@@ -1,8 +1,652 @@
+				"type": [ "string", "object" ],
+				"properties": {
+					"version": {
+						"type": "string"
+					},
+					"type": {
+						"type": "string",
+						"default": "default",
+						"enum": [ "default", "build" ]
+					}
+				}
+			}
+		}
+	}
+		
+For additional information about how dependency versions are chosen, see `Dependency Resolution <https://github.com/aspnet/Home/wiki/Dependency-Resolution>`_.
+		
+Package dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+The DNX Utility (dnu) is responsible for all operations involving packages in your application. You use the **Install** command to download a package based on the package id and add it to your application::
+
+	dnu install <package id>
+	
+For a list of usage, arguments, and options of the ``install`` command, enter the following in the command windows::
+
+	dnu help install
+	
+For more information about ``dnu`` commands, see `DNX Utility <https://github.com/aspnet/Home/wiki/DNX-utility>`_.
+
+The more common way of installing packages is to just edit the *project.json* file. Editors like Visual Studio provide IntelliSense for all packages, making editing the file far easier than running ``dnu install``. However, you can use ``dnu install`` if you prefer. 
+
+Assembly references
+^^^^^^^^^^^^^^^^^^^
+
+You can also specify a list of framework assemblies for frameworks other than *dnxcore50*:
+
+.. code-block:: json
+
+    {
+      "frameworks": {
+        "dnx451":{
+          "frameworkAssemblies":{
+            "System.Text": ""
+          }
+        }
+      }
+    }
+
+Generally, you use this frameworkAssemblies section when you want to depend on an assembly that is:
+- In the Global Assembly Cache (GAC)
+- Part of the framework you are targeting 
+
+The *frameworkAssemblies* section is separate from the rest of the dependencies list to remove the possibility of accidentally depending on a NuGet package that happens to have the same name as a .NET Framework assembly. 
+
+Project references
+^^^^^^^^^^^^^^^^^^
+
+The *global.json* file is used to configure the solution as a whole. It includes just two default sections, the ``projects`` section and the ``sdk`` section.
+
+.. code-block:: json
+
+	{
+	  "projects": [ "src", "test" ],
+	  "sdk": {
+		"version": "1.0.0-beta5",
+		"runtime": "clr",
+		"architecture": "x86"
+	  }
+	}
+
+The ``projects`` property designates which folders contain source code for the solution. By default, the project structure places source files in a *src* folder, allowing build artifacts to be placed in a sibling folder, making it easier to exclude such things from source control.
+
+Specifying required SDK version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``sdk`` property specifies the version of the DNX (.Net Execution Environment) that Visual Studio will use when opening the solution. It’s set in the *global.json* file, rather than in *project.json* file, to avoid scenarios where different projects within a solution are targeting different versions of the SDK.
```

Comment:
Can we explicitly mention here that it doesn't mean you will get the 'right' version when running `dnx web`? global.json doesn't impact the commandline and this causes confusion from time to time.